bndtools.core/src/bndtools/model/repo/RepositoryTreeContentProvider.java

```diff
@@ -125,7 +125,7 @@ public void inputChanged(Viewer viewer, Object oldInput, Object newInput) {
 
             // check to make sure there are actual changes to repo lists
             // before clearing cache
-            if (oldInput != null && (!oldInput.equals(newInput))) {
+            if (oldInput != null) {
```

Comment:
Hey Ray, thanks for tweaking this caching clearing.  It should fix the issue here:

https://github.com/bndtools/bndtools/issues/1888

However, should we update the code comments, since they don't really make sense now?  Also, should we stub in the 'corrected' way now, for checking the change counts (even if they are just no-ops)?

Reply:
good point! I'm also still considering an optimization for the repo check here... just don't have a good plan quite yet. Conversing with @bjhargrave about it...